<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OxygenConfigurationUI</class>
 <widget class="QWidget" name="OxygenConfigurationUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>378</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="KTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Border size:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>frameBorder</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="KComboBox" name="frameBorder"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Title alignment:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>titleAlignment</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="KComboBox" name="titleAlignment"/>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Button size:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>buttonSize</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="KComboBox" name="buttonSize"/>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QCheckBox" name="animationsEnabled">
         <property name="text">
          <string>Enable animations</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Fine Tuning</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Background style: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>blendColor</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KComboBox" name="blendColor"/>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QCheckBox" name="titleOutline">
         <property name="text">
          <string>Outline active window title</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QCheckBox" name="narrowButtonSpacing">
         <property name="text">
          <string>Use narrow space between decoration buttons</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="KComboBox" name="sizeGripMode"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="sizeGripModeLabel">
         <property name="text">
          <string>Extra size grip display: </string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>sizeGripMode</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="KComboBox" name="separatorMode">
         <item>
          <property name="text">
           <string>Never Draw Separator</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Draw Separator When Window is Active</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Always Draw Separator</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Separator display:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>separatorMode</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Shadows</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="Oxygen::ShadowConfigurationUi" name="inactiveShadowConfiguration">
           <property name="title">
            <string>Window Drop-Down Shadow</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Oxygen::ShadowConfigurationUi" name="activeShadowConfiguration">
           <property name="title">
            <string>Active Window Glow</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="shadowSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Window-Specific Overrides</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="Oxygen::ExceptionListWidget" name="exceptions" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="_expertModeContainer" native="true">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="1">
       <widget class="QPushButton" name="_expertModeButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Oxygen::ShadowConfigurationUi</class>
   <extends>QGroupBox</extends>
   <header>oxygenshadowconfigurationui.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Oxygen::ExceptionListWidget</class>
   <extends>QWidget</extends>
   <header>oxygenexceptionlistwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>frameBorder</tabstop>
  <tabstop>titleAlignment</tabstop>
  <tabstop>buttonSize</tabstop>
  <tabstop>animationsEnabled</tabstop>
  <tabstop>_expertModeButton</tabstop>
  <tabstop>blendColor</tabstop>
  <tabstop>sizeGripMode</tabstop>
  <tabstop>separatorMode</tabstop>
  <tabstop>titleOutline</tabstop>
  <tabstop>narrowButtonSpacing</tabstop>
  <tabstop>inactiveShadowConfiguration</tabstop>
  <tabstop>activeShadowConfiguration</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
